juju/juju
# Lxd cert schema

## Description of change

> Why is this change needed?

Adding the certificate schema for the provider. This simple change
should allow access to the remote. Additionally we also change how
it looks up the cloud endpoint, so it allows a mix regional setup;
a cluster with different lxd servers, maybe local, remote etc.

## QA steps

> How do we verify that the change works?

Setup a remote lxd server and supply your own client certs along
with the server one, so that you can connect to the server.

## Documentation changes

> Does it affect current user workflow? CLI? API?

Nothing as yet, that will come later.
